src/Controller/Backend/Users.php

```diff
@@ -50,6 +50,10 @@ protected function addRoutes(ControllerCollection $c)
 
         $c->get('/roles', 'viewRoles')
             ->bind('roles');
+
+        $c->match('/invitation/{code}', 'invitation')
+            ->assert('code', '.*')
```

Comment:
`code` is required so this should be `.+` which is what it is by default so this line can be removed